### PR TITLE
doc: add tutorial for custom ssh key

### PR DESCRIPTION
**Pull request by atnanahidiw**
kalau edit doc perlu di PRin ga ya @fushar? ^^a

### CODE REVIEW

File: web/docs/deployment/installation.md [hunk 1]
```diff
@@ -56,6 +56,11 @@ In `vars.yml`, generate different random strings for these values:
 - `rabbitmq_password`
    * The password for the server and grader apps to connect to RabbitMQ.
 
+If you generate a different ssh key filename, edit the following variable in `vars.yml` to allow access to root:
+
+- `ansible_ssh_private_key_file`
```
Comment (fushar) [inline web/docs/deployment/installation.md]: I think it makes more sense if this is located in the "F. Running Ansible playbooks" section instead, what do you think?

Comment (atnanahidiw) [reply web/docs/deployment/installation.md]: sorry @fushar I just read your comment today ^^a

Comment (atnanahidiw) [reply web/docs/deployment/installation.md]: in terms of practicality, I think your suggestion is better
developer will encounter the error in step `F. Running Ansible playbooks`, and they can find the solution immediately
let me edit it 🙇

**Review by fushar — APPROVED** [review]
Thanks!

**Pull request merged**